### ISSUE TITLE
Yellow Ribbon Tool Page Content Feedback

### ISSUE THREAD

**Issue by stephorkandatx** [department-of-veterans-affairs/va.gov-team#9687]
## Issue Description
As a student veteran, I need to see the most accurate information (content) regarding the Yellow Ribbon program so that I am aware of and have the right expectation for eligibility. 

---
## Tasks
- [x] Review feedback from stakeholders that are recorded in this ticket for content recommendations and updates (additional feedback that may be provided in the future should be recorded in a separate ticket, provided it does not impact our deliverable for this specific ticket)
- [x] Recommended content updates are applied, via Drupal
- [x] Recommended content updates are applied, via FrontEnd changes (Kelson)
- [x] Recommended Drupal content updates are reviewed internally
- [x] If the internal review elicits requested modifications, these modifications are applied and reviewed
- [x] Recommended Drupal, Frontend content updates are published

## Acceptance Criteria
- [x] Updates created by the content team are reviewed and published
- [x] Feedback Number 1 recommendation (below) is applied to the site (Kelson)
---
## Feedback to review for updates and recommendations
1. Regarding this language: "If you already have Post-9/11 GI Bill benefits, the Yellow Ribbon Program can help you pay for tuition for an associate or higher degree." on https://www.va.gov/education/yellow-ribbon-participating-schools/
Feedback from our stakeholders explains that Institutions of Higher Learning (IHLs) may participate in the YR program. Along with associates degrees or higher programs, there are other programs that are permitted. High cost certificate programs (NCDs) are permitted to be covered by YR as long as they are offered by an IHL, and are included on the YR agreement. 
The school simply has to be classified as an IHL by the state education department. 
**Anne recommends we trim the sentence to: "If you already have Post-9/11 GI Bill benefits, the Yellow Ribbon Program can help you pay for tuition." (Kelson/ @kelsonic to fix in REACT tool.**

2. Regarding the section "And your school must meet certain requirements." on https://www.va.gov/education/about-gi-bill-benefits/post-9-11/yellow-ribbon-program/
Feedback from our stakeholders requests that we add information to the bullet list, and to also make the newly-added piece of information be in the color red
"All of these must be true. Your school: (same)
◾Is an Institution of Higher Learning and (new, and requests to be in red)
◾Offers the Yellow Ribbon Program, and (same)
◾Hasn’t offered the Yellow Ribbon benefit to more than the maximum number of students in their agreement with us, and (same)
◾Has certified your enrollment with us and provided Yellow Ribbon Program information. (same)
**Recommend we don't add this new bullet. a) institution of higher learning is a vague term to most, and to spell out what that means would belabor the bullets; b) The next bullet says "Offers the Yellow Ribbon Program," which means it will be easy enough for people to check to see if their school or program participates.**

3. The FAQ item about summer terms "Can I get Yellow Ribbon funds for summer terms?" on  https://www.va.gov/education/about-gi-bill-benefits/post-9-11/yellow-ribbon-program/
Feedback from our stakeholders: A school does not have to offer YR during the summer terms, even if they have funding remaining. They can choose to offer for only fall and spring as part of their YR policy.
**Anne to update the accordion.**

## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by AnneHurley** [comment]
@stephorkandatx @DanielleThierryUSDSVA @bethpotts  My thoughts on the three asks above:
1.	It may make sense to not call out any specific degrees/certificates at all. Simply change the wording to: The Yellow Ribbon Program can help you pay for tuition. 
2.	I wonder if we really need to add “Is an institution of higher learning” –  because I don’t know that most people would know what-all is included, and since we go on to list “Offers the YR program,” we can assume a school would not offer the program if it weren’t eligible to do so. Thoughts?

3.	I will rework the answer about summer terms.

**Comment by RLHecht** [comment]
@AnneHurley One fix to your change to number 1 above. Should it be "If you already have Post-9/11 GI Bill benefits, the Yellow Ribbon Program may be able to help you pay for tuition." Instead of using "can" in case there are schools that don't qualify.

**Comment by bethpottsVADEPO** [comment]
Hello! Thanks for your work on this @AnneHurley. And @kelsonic please still hold on making these changes for a bit.

Jen Lee has handed off to me communication with these stakeholders, so I'll send to Janice et al the following revision that Danielle and I worked up, taking the above ^ and the email this ticket was created from into account. I'm hoping this revision will address all of Janice's concerns: 

Find out if your school participates in the Yellow Ribbon Program. If you already have Post-9/11 GI Bill benefits, this program may help you pay for tuition for a degree or training program at a college or other institution of higher learning.

A couple other things:
- Let's go ahead and add Janice's bullet to the main YR page. I do agree with Anne H. that it's vague, but I also don't think it's confusing.
- Anne, when you're done updating No. 3 above, let me know! I'll take a look.
- The red text is just her way of calling out the change for our purposes. Janice wasn't suggesting adding the text to the site in red. So we don't need to explain to her that we don't use red text on the site, thank goodness!!

**Comment by AnneHurley** [comment]
@bethpottsVADEPO  Here are the 2 fixes to the main YR page: http://preview-prod.vfs.va.gov/preview?nodeId=936

and @kelsonic  - if you can add this new blurb to the top of the YR search tool itself, that would be great:

Find out if your school participates in the Yellow Ribbon Program. If you already have Post-9/11 GI Bill benefits, this program may help you pay for tuition for a degree or training program at a college or other institution of higher learning.

**Comment by bethpottsVADEPO** [comment]
@kelsonic Wait! Want to run it by stakeholders before you change the content on the tool. FYI, @AnneHurley

**Comment by kelsonic** [comment]
haha I'm going to wait on this until you all give me the ok, don't worry! 🙂

**Comment by bethpottsVADEPO** [comment]
@AnneHurley As we discussed, here are my edits to No. 3, for the accordion on the eligibility page:

This depends on if your school offers Yellow Ribbon benefits in the summer. Please check with your school.

**Comment by AnneHurley** [comment]
@RLHecht  This is ready for your copyedit/review and then publish: http://preview-prod.vfs.va.gov/preview?nodeId=936
@stephorkandatx  and @bethpotts FYSA

**Comment by RLHecht** [comment]
@AnneHurley This page has been published.

**Comment by stephorkandatx** [comment]
Waiting on stakeholder feedback - will move forward if we do not hear from them shortly. Additional feedback to be written up in new tickets

**Comment by kelsonic** [comment]
@bethpotts Awesome! Should this content remain after searching? Currently there are 2 content states (before and after searching for the first time).

**Comment by bethpottsVADEPO** [comment]
I see, that content goes away. Let's keep it that way. New content simply replaces the paragraph on the page in the before state.

**Comment by kelsonic** [comment]
Perfect, sounds good! Thanks! 🙂

**Comment by kelsonic** [comment]
PR merged! Ready for validation following today's deployment 🙂

**Comment by bethpottsVADEPO** [comment]
Validated!
